components/collective-page/CollectiveNotificationBar.js

```diff
@@ -83,6 +83,16 @@ const messages = defineMessages({
     id: 'event.over.sendMoneyToParent.description',
     defaultMessage: 'Spend it by submitting event expenses, or transfer the remaining balance to the main budget.',
   },
+  tooFewAdmins: {
+    id: 'collective.tooFewAdmins',
+    defaultMessage:
+      'You need {missingAdminsCount, plural, one {one more admin} other {# more admins} } before you can accept financial contributions.',
+  },
+  tooFewAdminsDescription: {
+    id: 'collective.tooFewAdmins.description',
+    defaultMessage:
+      'Your collective doesn’t meet the requirements of having a minimum of {numberOfAdmins, plural, one {# administrator} other {# administrators} }. Add more administrators to comply with your host’s policy.',
```

Comment:
Just my opinion here, but I feel like we're losing an opportunity to inform the user that:
- If they already invited an admin, they just need to wait until the invited admin accepts the invitation;
- When the invited admin accepts the invitation, the collective will be automatically able to accept contributions.

Reply:
Good point, will try to adjust this message!

Reply:
Updated message:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/1552194/197163882-10bcdf5c-eca2-495c-878b-9bb302d4066b.png">